packages/caliper-samples/network/fabric-v1.4.1/swarm1org3peercouchdb/docker-compose.yaml

```diff
@@ -0,0 +1,408 @@
+version: '3'
+
+networks:
+  caliper-nw:
+    external: true
+
+services:
+  ca-org1:
+    image: hyperledger/fabric-ca:1.4.1
+    environment:
+      - FABRIC_CA_HOME=/etc/hyperledger/fabric-ca-server
+      - FABRIC_CA_SERVER_CA_NAME=ca.org1.example.com
+    ports:
+      - 7054:7054
+    command: sh -c 'fabric-ca-server start --ca.certfile /etc/hyperledger/fabric-ca-server-config/ca.org1.example.com-cert.pem --ca.keyfile /etc/hyperledger/fabric-ca-server-config/key.pem -b admin:adminpw -d'
+    volumes:
+      - /tmp/config-swarm/crypto-config/peerOrganizations/org1.example.com/ca/:/etc/hyperledger/fabric-ca-server-config
+    networks:
+      - caliper-nw
+    deploy:
+      replicas: 1
+      placement:
+        constraints:
+          - node.hostname==docker-desktop
```

Comment:
Just a question. If this is changed to another ip, it will be placed in the location ?

Reply:
I think IP address can not be specified. You can change this to any hostname listed in 'docker node list' or you can use predefined roles or user-defined tags as placement rules.